target/linux/pistachio/dts/pistachio_marduk_cascoda.dts

```diff
@@ -0,0 +1,352 @@
+/*
+ * Copyright (C) 2015 Imagination Technologies Ltd.
+ * Copyright (C) 2015 Google, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+/dts-v1/;
+
+#include "pistachio.dtsi"
+#include <dt-bindings/sound/pistachio-audio.h>
+#include "pistachio_board_uboot.dtsi"
+
+/ {
+	model = "marduk - Ci40 with CC2520";
```

Comment:
I would think you want to change the name here if it uses cascoda instead of cc2520